spajoa.py

```diff
@@ -43,7 +43,7 @@ def run_optimal_737(objective = 'fuel'):
         })
     else:
         substitutions.update({
-           'n_{pax}': [180.],
+           'n_{pax}': 180.,
            'ReqRng': 3000.*units('nmi'),
```

Comment:
Sorry about this. I was on a previous version of gpkit, and so it wasn't running _with_ the brackets for me. Odd.